app/models/User.js

```diff
@@ -1247,124 +1198,17 @@ module.exports = (User = (function () {
       return value
     }
 
-    getHackStackV2ExperimentValue () {
```

Comment:
Need to cleanup here: https://github.com/codecombat/hackstack/blob/main/src/lib/utils.ts#L422 or revert this change